$NetBSD$

The original sed script eats space chars around '-lpthread',
getting g++: unrecognized option '-pthread-lrt'.
Thus work arround:

--- configure.orig	2012-10-21 05:31:25.000000000 +0900
+++ configure	2012-12-25 20:10:40.816648000 +0900
@@ -9938,8 +9938,8 @@
           ac_cv_xmlrpc=yes
 
           test "x$XMLRPC_CFLAGS" = "x" && XMLRPC_CFLAGS=`$XMLRPC_C_CONFIG c++2 abyss-server --cflags`
           if test "x$XMLRPC_LIBS" = "x"; then
-              XMLRPC_LIBS=`$XMLRPC_C_CONFIG c++2 abyss-server --ldadd | sed s/.-lpthread.//`
+              XMLRPC_LIBS=`$XMLRPC_C_CONFIG c++2 abyss-server --ldadd | sed 's/.-lpthread./ /'`
               test "$ac_cv_static" = "yes" && XMLRPC_LIBS="-Wl,-Bstatic $XMLRPC_LIBS -Wl,-Bdynamic"
           fi
       fi

